collection-scripts/gather_network_logs_basics

```diff
@@ -153,6 +153,11 @@ if [[ "${CNCC_DEPLOYMENT}" == "cloud-network-config-controller" ]]; then
   oc adm inspect ${log_collection_args} --dest-dir must-gather cloudprivateipconfigs.cloud.network.openshift.io
 fi
 
+ROUTE_ADVERTISEMENTS=$(oc get network.operator.openshift.io cluster -o=jsonpath='{.spec.defaultNetwork.ovnKubernetesConfig.routeAdvertisements}')
```

Comment:
shouldn't this go into the check for only if network plugin is ovnk? i think we still support 3rd party plugins? but there we wouldn't have the .ovnkConfig set so value will be blank is it?